README.md

```diff
@@ -82,6 +86,11 @@ Optionally, you can specify the install location by providing the desired instal
 ```
 cmake .. -DCMAKE_INSTALL_PREFIX=/install/path
 ```
+On Mac OS X, you may also need to turn off `libprocps` and/or provide the path to openssl (your exact path may vary):
```

Comment:
We've now changed the default flag to have WITH_PROCPS=off, so I think this part of the change can be removed!

Reply:
In that case we can clarify what this flag does somewhere

Reply:
Good idea. Perhaps we should have a dedicated INSTALL.md file that details setting up the library if your not on mac os, and all the optional flags.